CDP4Composition/Converters/PanelViewConverter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
```

Comment:
Missing file header and using statements we put inside the namespace. For the header you can use ReSharper settings (i think it should already be setup with the repository) and using the Resharper cleanup command (you have to see in your Resharper settings the different levels of cleanup).